GEOSldas_App/sample_config_files/LADAS/exeinp.txt.Hy4dEnVar.atmens

```diff
@@ -11,14 +11,17 @@
 #
 ##############################################################################
 
-NUM_LDAS_ENSEMBLE: [NUM_ATM_ENSEMBLE]
+# e.g., ATMENS_AGCM_GRID = CF0090x6C
+EXP_DOMAIN:        [ATMENS_AGCM_GRID]_GLOBAL
+
+NUM_LDAS_ENSEMBLE: [NUM_ATMENS_ENSEMBLE]
 
 LADAS_COUPLING:    2
 
 ADAS_EXPDIR:       [full_path]/[ADAS_EXPDIR]
 
-MET_TAG:           [ADAS_EXPID]__Nx+- 
-MET_PATH:          [ADAS_EXPDIR]/atmens/ensdiag/mem
+MET_TAG:           [ADAS_EXPID]__bkg 
```

Comment:
When we configure LDAS from within ADAS (fvsetup), ADAS_EXPID should be passed into ldas_setup via the new command-line arguments.  I assume (but don't know for sure) that we can then leave this blank in the exe_input file.  In any case, we need to add comments into this sample exe_input file that clarify which resource parameters need to be specified and which resource parameters are provided via fvsetup.

Reply:
ADAS_EXPID and ADAS_EXPDIR are inputed from fvsetup to  ldas_setup, but I have not tested to leave the entries  blank in exeinp ( only tested  those set with some default strings ).  I will add a list of  parameters provided by fvsetup in the comments in the txt file, and it will be included in updating LADAS instruction on wiki.

Reply:
@saraqzhang : were you able to test what happens when the rc variables that are provided by fvsetup are left blank in the LDAS.rc file?

Reply:
I tested ADAS_EXPID left blank, it works, in ldas_setup it is a keyword "excluded from LDAS.rc".  But there are other variables indirectly rely on ADAS input, such as EXP_ID ( $ADAS_EXPID + '_LDAS' ),  it cannot be left blank, in ldas_setup  it is "Required exe input fields".

Reply:
correction:  tested ADAS_EXPDIR  left blank ( ADAS_EXPID is extracted from that) .

Reply:
Thanks, @saraqzhang.   It's probably best for now to keep the placeholders in the sample exeinp file.  At a later time, we can clean up what is and isn't required in the exeinp file to accommodate the new ldas_setup run mode (when fvsetup calls ldas_setup).  @weiyuan-jiang will look at the changes in ldas_setup to make sure they're consistent with the overall design of ldas_setup, which may (or may not) entail additional clean-up.